app/src/main/java/com/nervousfish/nervousfish/activities/MainActivity.java

```diff
@@ -225,14 +243,43 @@ private void fillDatabaseWithDemoData() throws IOException {
     }
 
     /**
+     * Called when a Bluetooth connection is established.
+     *
+     * @param event Contains additional data about the event
+     */
+    @Subscribe(threadMode = ThreadMode.MAIN)
+    public void onNewBluetoothConnectedEvent(final BluetoothConnectedEvent event) {
+        LOGGER.info("onNewBluetoothConnectedEvent called");
+
+        final Intent intent = new Intent(this, WaitActivity.class);
+        intent.putExtra(ConstantKeywords.SERVICE_LOCATOR, this.serviceLocator);
+        intent.putExtra(ConstantKeywords.WAIT_MESSAGE, getString(R.string.wait_message_slave_verification_method));
+        this.startActivityForResult(intent, ConstantKeywords.START_RHYTHM_REQUEST_CODE);
+    }
+
+    /**
      * Called when a new contact is received
      *
      * @param event Contains additional data about the event
      */
     @Subscribe(threadMode = ThreadMode.MAIN)
     public void onNewDataReceivedEvent(final NewDataReceivedEvent event) {
         LOGGER.info("onNewDataReceivedEvent called");
-        if (event.getClazz().equals(Contact.class)) {
+        if (event.getClazz().equals(String.class)) {
+            final String verificationMessage = (String) event.getData();
+
+            if ("rhythm".equals(verificationMessage)) {
+                //Go to RhythmActivity
+                final Intent intent = new Intent(this, RhythmCreateActivity.class);
+                intent.putExtra(ConstantKeywords.SERVICE_LOCATOR, serviceLocator);
+                this.startActivityForResult(intent, 0);
+            } else if ("visual".equals(verificationMessage)) {
```

Comment:
Could you change this if-statements from `"method".equals(verificationMessage)` to `verificationMessage.equals("method")` 😇 

Also, the comments `//Go to the ...Activity` seem a bit unnecessary to me...